jepsen/src/jepsen/nemesis/combined.clj

```diff
@@ -249,6 +263,133 @@
                              :fs    #{:strobe-clock}
                              :color "#A0E9E3"}}})))
 
+(defn wan-nemesis
+  "A nemesis which simulates running the cluster on a wide-area network, e.g.
+  a cluster of nodes split across multiple different AWS regions, i.e. higher
+  average latency, larger latency variance and spottier connectivity.
+
+  This nemesis responds to the following f's:
+
+      :drop     (Drop traffic between two nodes.)
+      :slow     (Delays network packets.)
+      :flaky    (Introduces randomzied packet loss.)
+      :heal     (End all traffic drops and restore network to fast operation.)
+      :fast     (Removes packet loss and delays.)"
```

Comment:
Likewise, for all of these, I want to see if we can (where meaningful) preserve symmetry with the other nemesis operations in this nemesis. That might not make sense; I'm not fully sure yet. Might also require expanding the net protocol, which I'm happy to do.